### PR TITLE
Change export type and file name for exporting data

### PR DESCRIPTION
**Pull request by yuanchen233**
Rename participant groups to deployments when exporting data
Add messages for errors when generating exports

### CODE REVIEW

File: src/test/kotlin/dk/cachet/carp/webservices/export/command/ExportAnonymousParticipantsTest.kt
Comment (davidscavnicky) [inline src/test/kotlin/dk/cachet/carp/webservices/export/command/ExportAnonymousParticipantsTest.kt:1]: check versioning of grade dependencies; check what mockk was used before in project. If


WARNING: Failed to transform class dk/cachet/carp/common/infrastructure/services/ApplicationServiceDecorator
java.lang.reflect.MalformedParameterizedTypeException: Mismatch of count of formal and actual type arguments in constructor of dk.cachet.carp.common.infrastructure.services.ApplicationServiceDecorator$invokeService$1: 0 formal argument(s) 2 actual argument(s)

**Review by davidscavnicky — COMMENTED** [review]
commented in test cases

File: src/test/kotlin/dk/cachet/carp/webservices/export/command/ExportAnonymousParticipantsTest.kt
Comment (yuanchen233) [reply src/test/kotlin/dk/cachet/carp/webservices/export/command/ExportAnonymousParticipantsTest.kt:1]: It's complaining about not using super when constructing super class, while we are mocking the object and Mockk package is handling this situation.

Comment (davidscavnicky) [reply src/test/kotlin/dk/cachet/carp/webservices/export/command/ExportAnonymousParticipantsTest.kt:1]: understood there's a ticket for it

**Review by davidscavnicky — COMMENTED** [review]
the warning issue has to do with dependencies, the ticket is there. no need to make changes at the moment

**Review by SlimShadyIAm — APPROVED** [review]
LGTM 👍

**Pull request merged**